commands/geosearch.md

```diff
@@ -0,0 +1,31 @@
+GEOSEARCH is compatible with `GEORADIUS`, in addition to searching for circular areas, it can also support searching for rectangular areas.
+
+The search center point is determined by the following option:
+
+* `FROMMEMBER`: It takes the name of a member already existing inside the geospatial index represented by the sorted set.
+* `FROMLOC`: Use parameters to pass longitude and latitude.
+
+The search area is determined by the following options:
+
+* `BYRADIUS`: Similar to GEORADIUS, searching for circular areas according to radius.
+* `BYBOX`: Axis-aligned rectangle, determined by height and width.
+
+`WITHCOORD`, `WITHDIST`, `WITHHASH`, `COUNT`, `ASC|DESC` option can refer to `GEORADIUS` commands.
```

Comment:
considering we're gonna deprecate GEORADIUS sooner or later, we probably wanna copy the description here rather than refer to it there.
maybe not just yet though.

Reply:
done, copy it.